CHANGELOG.md

```diff
@@ -89,6 +90,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ### Bug Fixes
 
+* [#2086](https://github.com/osmosis-labs/osmosis/pull/2086) `ReplacePoolIncentivesProposal` ProposalType() returns correct value of `ProposalTypeReplacePoolIncentives` instead of `ProposalTypeUpdatePoolIncentives`
```

Comment:
Do we need a second entry for this?

Reply:
I've manual tested this before on a local one node testnet, I'll try to submit proposals on osmosis testnet just for sure

Reply:
Thank you. If this is manually tested on the local net, we can merge.